src/device-config.ts

```diff
@@ -118,7 +118,7 @@ const configKeys: Dictionary<ConfigOption> = {
 	appUpdatePollInterval: {
 		envVarName: 'SUPERVISOR_POLL_INTERVAL',
 		varType: 'int',
-		defaultValue: '60000',
+		defaultValue: '900000',
```

Comment:
@Page- do you know of a reason this value was set to 60 seconds instead of the 15 minutes we set on the API?